aspnetcore/host-and-deploy/iis/hosting-bundle.md

```diff
@@ -59,6 +59,7 @@ To obtain an earlier version of the installer:
 
 > [!NOTE]
 > For information on IIS Shared Configuration, see [ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration).
+> When running the Hosting Bundle installer with options set, the value for each option will be saved in the registry. Subsequent installs from the same Major.Minor version band will use the same options, unless another set of options is explicitly passed from the command line. If the first install of the hosting bundle has no options passed, each option will get a default value of `0` written in to the registry.
```

Comment:
This looks good. Could we also articulate what writing a default value of 0 implies (option command line option is treated as on or off)?

Reply:
Updated